### PR TITLE
SSCS_WA_1_0:WA FTs

### PR DESCRIPTION
**Pull request by abul-syed**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/AM-2858

### Change description ###

Fix failing FTs

- [x] `f-004 31` - issue with order of list RESOLVED by `"__elementId__": "roleName,attributes.jurisdiction"`
- [x] `f-004 32` - issue with order of list RESOLVED by `"__elementId__": "roleName,attributes.jurisdiction"`
- [ ] `f-005 42` - failing as it's calling AAT RAS. hopefully should pass when RAS is updated to use hmcts/am-role-assignment-service#1932
- [ ] `f-005 43` - same issue as `f-005 42`, should pass when RAS is updated.
- [x] `f-005 44` - RESOLVED - added missing workTypes
- [x] `f-006 51` - RESOLVED - added missing workTypes
- [ ] `f-006 52` - same issue as `f-005 42`, should pass when RAS is updated.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -76,7 +76,9 @@ env.PACT_BROKER_FULL_URL = "https://pact-broker.platform.hmcts.net"
 env.AMQP_HOST = "rd-servicebus-aat.servicebus.windows.net"
 env.CRD_TOPIC_NAME = "rd-caseworker-topic-aat"
 env.JRD_TOPIC_NAME = "rd-judicial-topic-aat"
-env.ROLE_ASSIGNMENT_URL = "http://am-role-assignment-service-aat.service.core-compute-aat.internal"
+//env.ROLE_ASSIGNMENT_URL = "http://am-role-assignment-service-aat.service.core-compute-aat.internal"
+// TODO - change below back to aat http://am-role-assignment-service-aat.service.core-compute-aat.internal
+env.ROLE_ASSIGNMENT_URL = "https://am-role-assignment-service-pr-1932.preview.platform.hmcts.net"
```
Comment (mattnayler) [inline Jenkinsfile_CNP:81]: RAS dependency workarround to be removed in before merging to `master`.

File: charts/am-org-role-mapping-service/values.preview.template.yaml [hunk 2]
```diff
@@ -30,7 +30,9 @@ java:
     S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     CASE_WORKER_REF_APP_URL: http://rd-caseworker-ref-api-aat.service.core-compute-aat.internal
     JUDICIAL_REF_APP_URL: http://rd-judicial-api-aat.service.core-compute-aat.internal
-    ROLE_ASSIGNMENT_APP_URL: http://am-role-assignment-service-aat.service.core-compute-aat.internal
+#    ROLE_ASSIGNMENT_APP_URL: http://am-role-assignment-service-aat.service.core-compute-aat.internal
+#    TODO - change back ROLE_ASSIGNMENT_APP_URL to http://am-role-assignment-service-aat.service.core-compute-aat.internal
+    ROLE_ASSIGNMENT_APP_URL: http://am-role-assignment-service-pr-1932-java
```
Comment (mattnayler) [inline charts/am-org-role-mapping-service/values.preview.template.yaml:35]: RAS dependency workarround to be removed in before merging to `master`.

File: src/functionalTest/resources/features/F-004/S-031.td.json [hunk 3]
```diff
@@ -9,6 +9,10 @@
   "method": "POST",
   "uri": "{{ROLE_ASSIGNMENT_URL}}/am/role-assignments/query",
   "request": {
+    "headers": {
+      "sort": "roleName",
+      "direction": "ASC"
+    },
```
[the remark this answers is not in the document]
Comment (abul-syed) [reply src/functionalTest/resources/features/F-004/S-031.td.json]: actioned. see e6e3090

File: src/functionalTest/resources/features/F-004/S-032.td.json [hunk 4]
```diff
@@ -9,6 +9,10 @@
   "method": "POST",
   "uri": "{{ROLE_ASSIGNMENT_URL}}/am/role-assignments/query",
   "request": {
+    "headers": {
+      "sort": "roleName",
+      "direction": "ASC"
+    },
```
[the remark this answers is not in the document]
Comment (abul-syed) [reply src/functionalTest/resources/features/F-004/S-032.td.json]: actioned. see e6e3090

**Review by mikebrownccd — APPROVED** [review]
LGTM

**Pull request merged**